set_configmap.ps1

```diff
@@ -1,10 +1,10 @@
 $text = [string]::Format(((Get-Content $env:ACTION_PATH/octopus_configmap.yaml) -join "`n"),
```

Comment:
If charts dir doesn't exist, will it fail?

Reply:
It will, why would there be use case of not having charts directory. As far as i have seen in our repos, we keep all our helm files inside charts directory. I was thinking thats the directory structure we use to keep the helm files.

Reply:
That's okay. There can be instances where that won't be the case.